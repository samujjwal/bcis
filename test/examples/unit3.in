unit mile
unit hour
unit mph = mile / hour
print(55*mph)
