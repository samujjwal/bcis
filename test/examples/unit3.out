55.0*mile*hour^-1
